collector/collector.go

```diff
@@ -401,12 +402,23 @@ func (e *Exporter) connect() error {
 	// if TNS_ADMIN env var is set, set ConfigDir to that location
 	P.ConfigDir = e.configDir
 
-	if strings.ToUpper(e.config.DbRole) == "SYSDBA" {
-		P.IsSysDBA = true
-	}
-
-	if strings.ToUpper(e.config.DbRole) == "SYSOPER" {
-		P.IsSysOper = true
+	switch strings.ToUpper(e.config.DbRole) {
+	case "SYSDBA":
```

Comment:
This is purely a style recommendation:
1. Create a new type for this rather than using string literals
2. switch on the new type

Reply:
I'd also segregate database role handling code to it's own function/file. gonna suggest `database_role.go`

Reply:
updated to use the AdminRole type from godror, which handles parsing the string, deals with unknown values correctly, etc.